app/src/main/java/org/simple/clinic/selectcountry/SelectCountryScreen.kt

```diff
@@ -171,6 +177,14 @@ class SelectCountryScreen(
   }
 
   override fun goToNextScreen() {
-    // TODO(vs): 2019-11-07 Open TheActivity
+    // This navigation should not be done here, we need a way to publish
+    // an event to the parent activity (maybe via the screen router's
+    // event bus?) and handle the navigation there.
+    // TODO(vs): 2019-11-07 Move this to an event that is subscribed in the parent activity
```

Comment:
Consider this before we implement communication inside a hierarchy? https://github.com/uber/RIBs/wiki#communication-between-ribs

Reply:
Discussed offline: our current screen navigation component does not full support the callback style upstream communication in some cases. We will need to rethink this, punting on it for later.